examples/hybrid/driver.jl

```diff
@@ -190,6 +192,8 @@ function additional_tendency!(Yₜ, Y, p, t)
     microphy_0M && zero_moment_microphysics_tendency!(Yₜ, Y, p, t)
     rad_flux && rrtmgp_model_tendency!(Yₜ, Y, p, t)
     has_turbconv && TCU.sgs_flux_tendency!(Yₜ, Y, p, t)
+    model_spec.non_orographic_gravity_wave &&
+        gravity_wave_tendency!(Yₜ, Y, p, t)
```

Comment:
I think this is the issue: this is a closure over a global non-constant variable. Let's fix this. Here, I'll push a commit